components/Tabs/BUILD

```diff
@@ -173,18 +179,37 @@ mdc_objc_library(
     ],
 )
 
+mdc_objc_library(
+    name = "unit_test_tabbarview_sources",
+    testonly = 1,
+    srcs = native.glob([
+        "tests/unit/TabBarView/*.m",
+    ]),
+    sdk_frameworks = [
+        "XCTest",
+    ],
+    deps = [
+        ":TabBarView",
+    ],
+)
+
 mdc_unit_test_suite(
     name = "unit_tests",
     deps = [
         ":unit_test_sources",
+        ":unit_test_tabbarview_sources",
         ":unit_test_theming_sources",
     ],
 )
 
 mdc_snapshot_objc_library(
     name = "snapshot_test_lib",
-    extra_srcs = native.glob(["tests/snapshot/Theming/*.m"]),
+    extra_srcs = native.glob([
```

Comment:
Should we be using `native.glob` and not just `glob`?

Reply:
`native.glob` isn't prohibited in bazel 0.20 and this matches our current convention. Perhaps it is banned now (we're really outdated), but this isn't the PR to switch our conventions.

Reply:
I've opened [an issue to track updating our code base's BUILD files to address this concern](https://github.com/material-components/material-components-ios/issues/7639).